doc/sphinxman/source/occ.rst

```diff
@@ -370,6 +389,10 @@ preference to the default module, issue ``set qc_module occ``.
 
 Starting in v1.4, MP2.5 and MP3 default to the density-fit algorithm. Set |globals__mp_type| to ``CONV`` to get previous behavior.
 
+Publications resulting from the use of the non-OO CC codes should cite the following publications:
```

Comment:
```suggestion
Publications resulting from the use of the REMP codes should cite the following publications:
```

I'm very confused why this said "non-OO CC" originally. Am I missing something?

Reply:
yes, there's a list of all the other methods and citations that I didn't pick into this PR. Probably best to consider docs as a whole later on.